### ISSUE TITLE
The Aarogya Setu App is registering a mobile number without any verification of the mobile number owner.

### ISSUE THREAD

**Issue by mailtoPinkyG** [nic-delhi/AarogyaSetu_Android#394]
Bug of Aarogya Setu App
The Aarogya Setu App is registering a mobile number without any verification of the mobile number owner.In my case, somebody called 'Abhay Naraini' tested Corona positive in some lab and he gave a false mobile number (which is mine) and App registered and recommended me as "Under Medical Assistance " and I am being approached from the authorities just because this BUG!!!

**Comment by Bhartendu-Kumar** [comment]
Can you elaborate on the location details of Mr. Abhay Narani and your own?

**Comment by mailtoPinkyG** [comment]
Yes Sreejeet , Good idea to include verification system like OTP..
